test/index.js

```diff
@@ -57,12 +57,12 @@ describe('promise resolution', function () {
 
     resolver({
       foo: p1.promise,
-      bar: { baz: p2.promise }
+      bar: {baz: p2.promise}
     }, function (err, res) {
       if (err) return done(err)
       res.should.have.property('foo', 'bar')
       res.should.have.property('bar')
-        .and.have.property('baz', 'buz')
+          .and.have.property('baz', 'buz')
```

Comment:
sorry, my editor defaults to 4-space indent. I added a .editorconfig file to straighten it out (http://editorconfig.org/)